### PR TITLE
Circularity kit update 24.08 adoption view

### PR DESCRIPTION
**Pull request by LuisRickert**
### Added

- add new figures
- add license files for new figures

### Changed

- change strucutre of the contents. Same content chapters are in different order
- change wording

[Issue 703](https://github.com/eclipse-tractusx/sig-release/issues/703)

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: docs-kits/kits/Circularity_KIT/page-adoption-view.md [hunk 1]
```diff
@@ -81,6 +89,238 @@ Benefits for OEMs, SMEs, and Solution Providers:
...
+
+### Business Context
+
+#### Data Journey "End-of-Life"
+
+The "Data Journey" represents the entire process of dismantling an End-of-Life (EoL) vehicle. Each process step is accompanied by CX data models that would be necessary for digitally representing the individual process steps. This enables us to elevate the data journey for the dismantling process to a digital level and to gain an understanding of which data attributes and aspect models are required for each process step. It forms the foundation for the creation of digital EoL-solutions and the use of robotics.
+
+##### Figure 3
+
+![Data Journey - End of LIfe](resources/adoption-view/figure_dataJourney_EOF.svg)
```
Comment (catroest) [inline docs-kits/kits/Circularity_KIT/page-adoption-view.md:118]: This includes references to deprecated standards

**Comment by danielmiehle** [comment]
@LuisRickert : can you please update your dependencies?

**Comment by LuisRickert** [comment]
> @LuisRickert : can you please update your dependencies?

Which section are you referring to  ?

**Comment by jSchuetz88** [comment]
> > @LuisRickert : can you please update your dependencies?
> 
> Which section are you referring to ?

I assume Daniel refers to the 3rd Party dependency check that currently fails as a part of the automated check.

**Review by jSchuetz88 — APPROVED** [review]
Looks good to me, only the 3rd party dependencies needs to be updated.

**Comment by LuisRickert** [comment]
> @LuisRickert : can you please update your dependencies?

Done

**Review by maximilianong — APPROVED** [review]

**Review by danielmiehle — APPROVED** [review]

**Comment by arnoweiss** [comment]
@LuisRickert  - we have conflicts on this PR. Please check if it was superseeded by an already merged PR.

**Comment by jSchuetz88** [comment]
Hi @LuisRickert,

I'm confused, wasn't this branch already merged?

**Comment by jSchuetz88** [comment]
> > Hi @LuisRickert,
> > I'm confused, wasn't this branch already merged?
> 
> No it is still open, there was a merge conflict and there is still an IP approval pending. I just resolved the merge conflict. I am still waiting for the final approval that this can be published... . I expect the answer today
> 
> edit: typo, details

Sorry, I think is missed this one, as there was a second circularity kit PR which has been merged.

Could you add the IP-check ticket into the description?

**Comment by LuisRickert** [comment]
IP-Clearance Granted

**Comment by stephanbcbauer** [comment]
@jSchuetz88 can you check this PR, please? These changes are right now not in 24.08? Should they?

**Comment by stephanbcbauer** [comment]
> > > Hi @LuisRickert,
> > > I'm confused, wasn't this branch already merged?
> > 
> > 
> > No it is still open, there was a merge conflict and there is still an IP approval pending. I just resolved the merge conflict. I am still waiting for the final approval that this can be published... . I expect the answer today
> > edit: typo, details
> 
> Sorry, I think is missed this one, as there was a second circularity kit PR which has been merged.
> 
> Could you add the IP-check ticket into the description?

@LuisRickert couls you please add the IP Check issue in the comments/description? Thx

**Comment by stephanbcbauer** [comment]
@jSchuetz88 [IP Issue](https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/16324) created

**Comment by stephanbcbauer** [comment]
@jSchuetz88 is this still needed? @LuisRickert please resolve the conflicts. The related [IP Ticket](https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/16324) is still under review

**Comment by stephanbcbauer** [comment]
> @jSchuetz88 is this still needed? @LuisRickert please resolve the conflicts. The related [IP Ticket](https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/16324) is still under review
> 
> It seems there is a Multiple Headings error in the following file
> 
> ```shell
> docs-kits/kits/Circularity_KIT/page-adoption-view.md:114 MD024/no-duplicate-heading/no-duplicate-header Multiple headings with the same content [Context: "### Business Context"]
> ```

IP Tickets was approved and closed. Please resolve the conflicts

**Pull request closed without merging**